packages/coverage-istanbul/src/provider.ts

```diff
@@ -173,6 +180,35 @@ export class IstanbulCoverageProvider implements CoverageProvider {
       }
     }
   }
+
+  async includeUntestedFiles(coverageMap: CoverageMap) {
+    const files = await this.testExclude.glob(this.ctx.config.root)
+
+    for (const file of files) {
+      const filename = resolve(this.ctx.config.root, file)
+
+      if (!coverageMap.data[filename]) {
+        const transformResult = await this.ctx.vitenode.transformRequest(filename) as TransformResult
+        const sourceMap = transformResult?.map as NonNullable<TransformResult['map']>
```

Comment:
I'm not sure about the `this.ctx.vitenode.transformRequest` part. Just saw that this was used elsewhere to get source maps. 

Does this look alright? The goal here is to load transformed code and source maps for it. After quick testing this seemed to work OK.